pkgs/development/python-modules/textnets/default.nix

```diff
@@ -59,10 +63,22 @@ buildPythonPackage rec {
     "textnets"
   ];
 
+  # Enables the package to find the cythonized .so files during testing. See #255262
+  preCheck = ''
+    python setup.py build_ext --inplace
```

Comment:
```suggestion
    rm -r textnets
```
Please do not rebuild before checkPhase.

<summary>Build Log</summary>
<details>

```
textnets> Running phase: pytestCheckPhase
textnets> Executing pytestCheckPhase
textnets> 
textnets> [1/1] Cythonizing textnets/_ext.pyx
textnets> /nix/store/43rx85r69ymjxlw9xsg5j2xh2b7k2r5z-python3.11-cython-3.0.7/lib/python3.11/site-packages/Cython/Compiler/Main.py:381: FutureWarning: Cython directive 'language_level' not set, using '3str' for now (Py3). This has changed from earlier releases! File: /build/textnets-0.9.4/textnets/_ext.pxd
textnets>   tree = Parsing.p_module(s, pxd, full_module_name)
textnets> running build_ext
textnets> building 'textnets._ext' extension
textnets> gcc -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -I/nix/store/9109gq5pbiizq4jf8qcyrdxcc15chcr5-libxcrypt-4.4.36/include -fPIC -I/nix/store/sxr2igfkwhxbagri49b8krmcqz168sim-python3-3.11.8/include/python3.11 -c textnets/_ext.c -o build/temp.linux-x86_64-cpython-311/textnets/_ext.o
textnets> gcc -shared -L/nix/store/eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee-bzip2-1.0.8/lib -L/nix/store/eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee-expat-2.6.0/lib -L/nix/store/eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee-libffi-3.4.4/lib -L/nix/store/9109gq5pbiizq4jf8qcyrdxcc15chcr5-libxcrypt-4.4.36/lib -L/nix/store/eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee-mpdecimal-4.0.0/lib -L/nix/store/eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee-ncurses-6.4/lib -L/nix/store/eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee-openssl-3.0.13/lib -L/nix/store/eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee-sqlite-3.45.1/lib -L/nix/store/eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee-xz-5.4.6/lib -L/nix/store/eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee-zlib-1.3.1/lib -L/nix/store/lb1z8nckdwx4kmwy19znh1bl1p63wxhv-tzdata-2024a/lib -L/nix/store/eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee-gdbm-1.23/lib -L/nix/store/eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee-readline-8.2p10/lib -L/nix/store/eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee-bzip2-1.0.8/lib -L/nix/store/eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee-expat-2.6.0/lib -L/nix/store/eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee-libffi-3.4.4/lib -L/nix/store/9109gq5pbiizq4jf8qcyrdxcc15chcr5-libxcrypt-4.4.36/lib -L/nix/store/eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee-mpdecimal-4.0.0/lib -L/nix/store/eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee-ncurses-6.4/lib -L/nix/store/eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee-openssl-3.0.13/lib -L/nix/store/eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee-sqlite-3.45.1/lib -L/nix/store/eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee-xz-5.4.6/lib -L/nix/store/eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee-zlib-1.3.1/lib -L/nix/store/lb1z8nckdwx4kmwy19znh1bl1p63wxhv-tzdata-2024a/lib -L/nix/store/eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee-gdbm-1.23/lib -L/nix/store/eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee-readline-8.2p10/lib build/temp.linux-x86_64-cpython-311/textnets/_ext.o -L/nix/store/sxr2igfkwhxbagri49b8krmcqz168sim-python3-3.11.8/lib -o build/lib.linux-x86_64-cpython-311/textnets/_ext.cpython-311-x86_64-linux-gnu.so
textnets> copying build/lib.linux-x86_64-cpython-311/textnets/_ext.cpython-311-x86_64-linux-gnu.so -> textnets
textnets> ============================= test session starts ==============================
```
</details>

> or try removing textnets from CWD

`CWD` stands for Current Working Directory.
In this case, it seems to work fine.

Reply:
Oh, now I see! Yes, rm -r textnets works. (This was _supposed_ to be a sinple version bump. Hah.)